GeneratorSchema/Schema.cs

```diff
@@ -58,6 +58,7 @@ IEnumerator IEnumerable.GetEnumerator()
     // Based on http://json-schema.org/latest/json-schema-validation.html#rfc.section.5 and http://json-schema.org/draft-04/schema
     public class Schema
     {
+        public override string ToString() => Description;
```

Comment:
What is this for?

Reply:
To have something useful in the debugger instead of `{KhronosGroup.Gltf.Generator.Schema.Schema}`.

Reply:
I changed it to `ReferenceType ?? Title ?? Description` now. Would be better to deserialize `$id` but idk cba maybe NewtonSoft meddles with it like it does with `$ref`, requiring another file scoped search/replace.

Reply:
I'm not sure I understand. You want to see the string for the description for the object directly instead of expanding the object?

Reply:
Yes, it is good practice to always provide `ToString`, `DebuggerDisplay` or `DebuggerTypeProxy`.

Reply:
Sure, but it seems weird to put the description, title, or reference type as the string. Anyways, not a big deal either way.